en/docs/learn/api-gateway/passing-end-user-attributes-to-the-backend/passing-enduser-attributes-to-the-backend-using-jwt.md

```diff
@@ -125,7 +125,7 @@ if the <code>getClaims</code> method returns <code>{email:user1@wso2.com, gender
 <td><pre><code>apim.jwt.claim_dialect</code></pre></td>
 <td><div class="content-wrapper">
 <p>The dialect URI under which the user's claims are be looked for. Only works with the default value of the <code>apim.jwt.claims_extractor_impl</code> element defined above.</p>
-<p>The JWT token contains all claims define in the <code>apim.jwt.claim_dialect</code> element. The default value of this element is <code>http://wso2.org/claims</code>. To get a list of users to be included in the JWT, simply uncomment this element after enabling the JWT. It will include all claims in <code>http://wso2.org/claims</code> to the JWT token.</p>
+<p>The JWT token contains all claims that are defined in the <code>apim.jwt.claim_dialect</code> element. The default value of this element is <code>http://wso2.org/claims</code>. To get a list of users to be included in the JWT, simply uncomment this element after enabling the JWT. It will include all claims in <code>http://wso2.org/claims</code> to the JWT token.</p>
```

Comment:
To get a list of users to be included in the JWT, simply uncomment this element after enabling the JWT -> I don't think this gets the list of users. Instead it gets the users claims?

Reply:
@chamilaadhi , please comment.

Reply:
yeah that a mistake :D . need to fix it as claims

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/798/commits/ad35368762f90e93da8563604b42628e902ea243